content-review/impl/turnitin-oc/src/main/java/org/sakaiproject/contentreview/turnitin/oc/ContentReviewServiceTurnitinOC.java

```diff
@@ -1426,7 +1440,7 @@ private void handleSubmissionStatus(JSONObject submissionJSON, ContentReviewItem
 						Map<String, String> assignmentSettings = assignment.getProperties();
 						//set item retryTime to due date
 						cal.setTime(getDueDateRetryTime(assignmentDueDate));
-						if("true".equals(assignmentSettings.get("store_inst_index"))){
+						if(indexingEnabled && "true".equals(assignmentSettings.get("store_inst_index"))){
```

Comment:
is this check needed since you already have a quick return within `indexSubmission`